### ISSUE TITLE
Annotation aspect not called with version 2.1.0 because of includePaths

### ISSUE THREAD

**Issue by bgaillard** [goaop/framework#315]
Hi, I updated Go-Aop from version `2.0.0` to version `2.1.0` and my aspects are not called anymore. 

I found that the problems comes from the `includePaths` property I used, with version `2.0.0` I used the following configuration.

```php
// src/main/php/Application/Service/Services.php
->init(
    [
        'debug' => $diContainer->get('go.aspectKernel.debug'),
         'appDir' => __DIR__ . '/../..',
        'cacheDir' => $diContainer->get('datapath') . '/aop',
        'includePaths' => [
            __DIR__   // Folder 'src/main/php/Application/Service'
        ]
    ]
);
```

This does not work with version `2.1.0` of Go-Aop, but if I remove the `includePaths` configuration (i.e take all sources into account) then it works. 

The aspects I'm using are configured with a custom `Transactional` annotation.

```php
    /**
     * Template method which wraps a method invotation around a database transaction.
     *
     * @Around("@execution(Gomoob\Aop\Annotation\Transactional)")
     *
     * @param MethodInvocation $methodInvocation an object which represents the service method whichis called.
     *
     * @return mixed the return of the function associated to the join point.
     */
    public function wrapTransaction(MethodInvocation $methodInvocation)
    {
        return $this->doWrapTransaction($methodInvocation);
    }
```

Sample use in my code.

```php
// src/main/php/Application/Service/Impl/BrandService.php

use Gomoob\Aop\Annotation\Transactional;

class BrandService extends AbstractMultilingualService implements IBrandService
{
    /**
     * {@inheritdoc}
     *
     * @Transactional
     */
    public function save(IBrand $brand)
    {
        ...
    }
}
```

As you can see my `BrandService.php` file is inside the `src/main/php/Application/Service/Impl` folder, the `init` call of the Aspect Kernel has been done ins the `src/main/php/Application/Service/Services.php` file. So in `Services.php` the `APP_DIR` constant is associated to the `src/main/php/Application/Service` folder. 

Do you confirm this is a `2.1.0` bug ? Or do I have to change the configuration of my `includePaths` property ? 

Thanks

**Comment by lisachenko** [comment]
Hi, do you use framework on Windows platform? I have a guess, that 2.1.0 can contain a bug with directory matching on Windows.

**Comment by lisachenko** [comment]
Closed by inactivity. 

Anyway, there was an issue with `fnmatch` function #326, which was fixed in 2.1.1. You should try to update to see if your issue was fixed
